### ISSUE TITLE
allow simpler integration into symfony framework

### ISSUE THREAD

**Issue by SimonHeimberg** [krzysztofruszczynski/Krsc-Reports-PHPExcel-Framework#2]
Please add some methods to allow to create an Symfony Reply.

I'd prefer something similar to this:
```php
    $response = $reportService->createStreamedResponse($reportService->getReportWriter($xlObj));
    $response->headers->add($reportService->getHeaders());
```
instead of
```php
$reportService->createReport();
die()
```

get some inspiration from
* https://github.com/liuggio/ExcelBundle/blob/0a9a5b0/Factory.php#L79
 * https://github.com/EmchBerger/cube-common-bundle/blob/e1504a5f29fe2e66a5a4192d6effd555e6b6b364/src/FileFormat/ExcelConverter.php#L98

**Comment by krzysztofruszczynski** [comment]
implemented here: 7f8bd66 0821f22
Example usage:

$oService = new \KrscReports\Service(); // or via symfony service
$oService->setFileName('abc');
$response = new \Symfony\Component\HttpFoundation\StreamedResponse(function () use ($oService) {
                $oService->createReport()
});
$oService->getFileObject()->setSymfonyHeaders($response->headers);

**Comment by SimonHeimberg** [comment]
problem: when `$oService->setFileName()` is not called, the headers are sent anyway.

workaround: call `$oService->setFileName('php:://output')`

**Comment by krzysztofruszczynski** [comment]
Hi @SimonHeimberg ,
Yes, `setSymfonyHeaders()` should disable sending headers by `createHeaders()` method. This bug is fixed in v2.0.5 release. Thank you for this feedback.
